### ISSUE TITLE
Adjust docs for breaking code

### ISSUE THREAD

**Issue by chriscrutt** [vyperlang/vyper#2043]
## Simple Summary
either update the pip download so it matches github commits and readthedocs, or change readthedocs to match the pip download

## Motivation
This made it very confusing when certain things wouldn't compile- including the examples.

## Specification
when using `pip3 install vyper`, it installs the 0.1.0b17+commit.**0671b7b** version of vyper, while the docs are adjusted for a newer version of the 0.1.0b17 commit. This is a very active repo, so I wouldn't say you should immediately update the docs after every commit, but I believe the docs should follow either the pip3 version, or be adjusted for breaking changes. For example, uint(wei) is required when using numbers in relation to transactions in the 0671b7b commit but not supported unit in the newer commits (such as the newest one, 4afbb2b, as of writing).

## Copyright
Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/)

**Comment by fubuloubu** [comment]
The `latest` version of the docs uses whatever is in the repo, so  for now please use https://vyper.readthedocs.io/en/v0.1.0-beta.17/ for b17 docs

I agree that it should probably only point to whatever the last release was, but we're planning on making a change to our release process soon where this will not be an issue

**Comment by iamdefinitelyahuman** [comment]
It should be possible to configure readthedocs to default to "stable", which is the last tagged commit.

**Comment by fubuloubu** [comment]
Sorry, didn't mean to close this

**Comment by fubuloubu** [comment]
Yeah, we'll have to add `stable` after `v0.2.0` is released. Right now it points to `v0.0.4`
